src/Catalog.API/appsettings.json

```diff
@@ -23,13 +23,14 @@
   },
   "CatalogOptions": {
     "UseCustomizationData": false
+  },
+  "AI": {
```

Comment:
@stephentoub - did you intend to uncomment this block?

Reply:
I did. Does it break something?

Reply:
It doesn't appear to break anything. Using OpenAI still works for me without tweaking it.